moped-editor/package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "react-material-dashboard",
-  "author": "DeviasIO",
+  "name": "atd-moped-editor",
+  "author": "ATD Data & Technology Services",
   "licence": "MIT",
```

Comment:
Not opinionated, but should we change the license type to GPL or something like that? Just out of curiosity

Reply:
yeah, I'm also not opinionated but I think the standard me and @johnclary had discussed a while back was just public domain CC0 1.0 Universal. 

ex: https://github.com/cityofaustin/atd-dockless-dataviz/blob/master/LICENSE

Probably something we should get back into the habit of updating across our projects.

Reply:
yep, that's generally what i use but also often forget to add it. i didn't realize the creative commons license [was available in github's pre-configured list](https://choosealicense.com/licenses/cc0-1.0/), but it is now. nice!